core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -691,6 +691,15 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  private[spark] val KERBEROS_FILESYSTEM_RENEWAL_EXCLUDE =
```

Comment:
Since `HadoopFSDelegationTokenProvider` obtains the initial tokens and not just renewing them why do we call it `...renewal...`?

Reply:
It seems related to YARN behavior. YARN will try to renew delegation token when the job is submitted to YARN. For the customer case, the delegation token is already obtained when calling `spark-submit`. But YARN is unable to renew the DT and refuses the job. This config is for such case to avoid YARN to renew the DT.

Reply:
I see the point. On the other hand the other use-case is that Spark user lets Spark to obtain tokens and such case the added configuration has effect. In the mentioned case the name is misleading because it has effect on the initial obtain too.

Reply:
Do you mean with empty renewer `addDelegationTokens` doesn't work to obtain the token?

Reply:
I mean the code works because I've tested it. My suggestion is not to name the config variable `...renewal...` because it has effect in the initial obtain too. The code is good as-is and good job :)

Reply:
Sorry I am confused. :) So you said this configuration also affects initial obtain too. It means if users set a file system in the configuration, Spark won't obtain initial token for it? Or I misunderstood above comments?